client/src/reducers/websiteReducer.js

```diff
@@ -43,11 +46,16 @@ const websiteReducer = (state = INITIAL_WEBSITE_STATE, { payload, type }) => {
         title: payload.title ?? state.title,
         pages: payload.pages ?? state.pages,
         publicationOptions: payload.publicationOptions ?? state.publicationOptions,
+        layout: payload.layout ?? state.layout,
+        darkTheme: payload.darkTheme ?? state.darkTheme,
       };
     case UPDATE_PUBLICATION_OPTIONS:
       return { ...state, publicationOptions: payload };
     case UPDATE_WEBSITE_TITLE:
       return { ...state, title: payload };
+    case UPDATE_WEBSITE_TEMPLATE:
+      // Theme '2' is dark mode
```

Comment:
If we have already decided theme 2 to be the dark mode, can we just have `const themeOptions = ['light', 'dark', 'etc'];` in the api/src/config/website.js ? It's hard to know what is the theme by just looking at the number. 
Or even simpler, if we only have dark/ light, can we just use `darkMode: boolean ` to store the data?

Reply:
@pmyip3 I think the first alternative you suggested `['light', 'dark', 'etc']` (as an enum) is preferred over the boolean. This way we can extend colour themes in the future. Although if needed to get this PR through for User Acceptance Testing, I think sticking with `['1', '2']` wouldn't be too bad (I'll leave that to Kunj to decide).